doc/api/cli.md

```diff
@@ -156,6 +156,40 @@ Currently the support for run-time snapshot is experimental in that:
    a report in the [Node.js issue tracker][] and link to it in the
    [tracking issue for user-land snapshots][].
 
+### `--compile=path`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+> This feature is currently being designed and will still change.
```

Comment:
I think this should be one block, right?
```suggestion
> Stability: 1 - Experimental: This feature is currently being designed and will
> still change.
```

Reply:
done

Reply:
@aduh95 @ovflowd his causes https://github.com/nodejs/node/actions/runs/3297903190/jobs/5439316276#step:5:37
```
TypeError: Cannot read properties of undefined (reading 'value')
[37](https://github.com/nodejs/node/actions/runs/3297903190/jobs/5439316276#step:5:38)
    at file:///home/runner/work/node/node/tools/doc/stability.mjs:79:43
```
while running `make doc-only`, so I have removed this change.

Reply:
This is odd. I tested these changes on the infrastructure available on `nodejs.dev`, multiline line stability indexes should be supported. I am wondering what is wrong here.

As you can see here for example: https://raw.githubusercontent.com/nodejs/node/main/doc/api/documentation.md

What happens if you change to this:

```
> Stability: 1 - Experimental. This feature is currently being designed and will change.
```

Reply:
@ovflowd that unfortunately creates this warning when I run `make lint-md` - https://github.com/nodejs/node/actions/runs/3360669817/jobs/5570148053#step:7:14:
```sh
doc/api/single-executable-applications.md
Warning:   5:95  warning  Line must be at most 80 characters  maximum-line-length  remark-lint
⚠ 1 warning
```

Reply:
I mean that works only if I place it after the `source_link` comment. It fails only if I place it above that.

Reply:
Let's place it after then, no?

Reply:
All the existing docs place it before the comment. Don't know if it matters though. Moved it below the comment.